src/components/TestDetailsVrt/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Error, Wrapper, Details } from './style';
+import BaselineVrt from '../BaselineVrt';
+import { useGlobalContext } from '../../hooks/GlobalContext';
+import { useReportContext } from '../../hooks/ReportContext';
+import { IVrtData } from '../../transform';
+import { getVrtData } from './utils';
+import { TabsList, TabsRoot, TabsTrigger } from '../TestStepsView/style';
+import * as Tabs from '@radix-ui/react-tabs';
+import { isVRT } from '../../constants';
+
+const TestDetailsVrt: React.FC = () => {
+  const { environmentName, fileId, testId } = useReportContext();
+  const { environments } = useGlobalContext();
+  const vrtData: IVrtData = getVrtData(environments[environmentName], fileId, testId);
+  return (
+    <Wrapper>
+      <Details>
+        <TabsRoot defaultValue="tab1" vrt={isVRT()}>
```

Comment:
since this is a new component specially for VRT, do we really need `vrt` prop?

Reply:
I am using common TabRoot component. So we need prop here